src/redux/actions/userActions.js

```diff
@@ -184,11 +178,7 @@ export const authenticateUser = (token) => {
           "auth-token": token,
         },
       };
-      const resp = await axios.post(
-        "http://localhost:3001/api/users/authenticate",
-        {},
-        config
-      );
+      const resp = await axios.post(`${BASE_URL}/authenticate`, {}, config);
```

Comment:
im assuming the base url is like http://localhost:3000/api so here you're missing the /users part of the path